README.md

```diff
@@ -2207,6 +2213,90 @@ workbook.commit()
   });
 ```
 
+##### Streaming XLSX Reader
+
+The streaming XLSX workbook reader is available in the ExcelJS.stream.xlsx namespace.
+
+The constructor takes a required input argument and an optional options argument:
+
+| Argument              | Description |
+| --------------------- | ----------- |
+| input (required)      | Specifies the name of the file or the readable stream from which to read the XLSX workbook. |
+| options (optional)    | Specifies how to handle the event types occuring during the read parsing. |
+| options.entries       | Specifies whether to emit entries (`'emit'`) or not (`undefined`). Default is `'emit'`. |
+| options.sharedStrings | Specifies whether to cache shared strings (`'cache'`), emit them (`'emit'`) or skip them (`undefined`). Default is `'cache'`. |
```

Comment:
may I please you to give some more explanations what's mean cache? 
Is it keep in the worksheet model, and it's available to read later, isn't it?

Reply:
It's cached into the workbook instance and then, whenever a shared string is found inside a cell, it sets that string:
```js
                        if (sharedStrings) {
                          cell.value = sharedStrings[index];
                        } else {
                          cell.value = {
                            sharedString: index,
                          };
                        }
```

Reply:
I updated the README - do you think it's clearer now?